src/utils/appHelper.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable max-len */
```

Comment:
Don't disable lint max-len here, writing functions in a single line make them less readable, I would suggest to format them in normal way :)